examples/run_test.py

```diff
@@ -8,25 +8,21 @@ def verifyParams():
     if (nPoints > 16*1024 or nPoints < 1):
         print("ERROR: MAXIMUM NUMBER OF POINTS IS 16384.")
         print("EXITING.")
-        quit()
     else:
         print("VERIFIED NUMBER OF POINTS.")
     if (nScans < 1):
         print("ERROR: THERE MUST BE AT LEAST 1 SCAN.")
         print("EXITING.")
-        quit()
     else:
         print("VERIFIED NUMBER OF SCANS.")
     if (p90 < 0.065):
         print("ERROR: PULSE TIME TOO SMALL.")
         print("EXITING.")
-        quit()
     else:
         print("VERIFIED PULSE TIME.")
```

Comment:
pretty sure this example was used for developer purposes not sure what to do with it, so I leave it be